explore/src/main/scala/explore/targeteditor/TargetEnvEditor.scala

```diff
@@ -27,17 +27,18 @@ import lucuma.core.model.SiderealTarget
 import lucuma.core.model.Target
 import lucuma.core.model.User
 import lucuma.ui.reusability._
-import monocle.function.Index._
+import monocle.Optional
 import react.common.ReactFnProps
 import react.semanticui.elements.button._
 import react.semanticui.shorthand._
 import react.semanticui.sizes._
 
 final case class TargetEnvEditor(
```

Comment:
Maybe we should rename `TargetEnv` to something else? Or do we still handle that concept?

Reply:
I think it should probably be the AsterismEditor.